scene/main/node.cpp

```diff
@@ -2198,8 +2198,10 @@ void Node::_duplicate_and_reown(Node *p_new_parent, const Map<Node *, Node *> &p
 		ERR_EXPLAIN("Node: Could not duplicate: " + String(get_class()));
 		ERR_FAIL_COND(!obj);
 		node = Object::cast_to<Node>(obj);
-		if (!node)
+		if (!node) {
 			memdelete(obj);
+			ERR_FAIL_COND(!node);
```

Comment:
Doesn't this add the kind of warning that you've been working around in previous PRs?

Reply:
Yes, it would have created a warning, but I had no idea of the text of ERR_EXPLAIN and ERR_FAIL_COND provides a little more information than ERR_FAIL itself.
On second thought I copied ERR_EXPLAIN from above, so ERR_FAIL provides enough information.